harness_overwrite.py

```diff
@@ -0,0 +1,91 @@
+import sys
+from typing import Dict, Optional, Tuple
+
+
+def get_env(line: str) -> Optional[Tuple[str, str, str]]:
+    if line and line[0] not in (" ", "#") and "=" in line:
+        key, others = line.split("=", maxsplit=1)
+        key = key.strip()
+        if "#" in others:
+            val, extra = others.split("#", maxsplit=1)
+            val = val.strip()
+        else:
+            val, extra = others.strip(), ""
+        return key, val, extra
+
+
+def manualy_parse(env_file: str) -> Dict[str, str]:
```

Comment:
"manually" as opposed with using `dot_env` which I couldn't figure out how to use to read from only one env file and ignore all other env variables.